frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -263,3 +268,135 @@ it("clicking on the button only adds the email once", () => {
   expect(handleClose).toHaveBeenCalledTimes(1);
   expect(addEmail).toHaveBeenCalledTimes(1);
 });
+
+describe("edit action dialog helper file", () => {
```

Comment:
These are great tests! I was going to say they should probably be in "editActionDialogHelper.test" rather than "EditActionDialog.test", but its basically the same name :P

Reply:
I simply put the tests there, because these functions are basically used in _editActionDialog.jsx_ component.

Also as you mentioned, the file name is almost the same, so there is no confusion here.